packages/playground/src/views/farms.vue

```diff
@@ -169,12 +172,17 @@ watch(mixedFarmFilters, updateFarms, { deep: true });
 watch(filterFarmInputs, updateQueries, { deep: true });
 watch(page, updateQueries);
 watch(size, updateQueries);
+
+// The filters should reset to the default value again..
 const inputFiltersReset = (nFltrNptsVal: FilterFarmInputs) => {
```

Comment:
change the variable name nFltrNptsVal to something understandable.

Reply:
I renamed it, Please give it a look again :)